Update PRAS version used in ReEDS to 0.8.0 (#47)
ReEDS-Model/ReEDS: docs/source/setup.md

```

### Repository Setup

> The ReEDS source code is hosted on GitHub: [https://github.com/ReEDS-Model/ReEDS-2.0](https://github.com/ReEDS-Model/ReEDS-2.0)

Reviewer: ```suggestion
The ReEDS source code is hosted on GitHub at <https://github.com/ReEDS-Model/ReEDS>.
```

#### Step 1: Install juliaup

````{tab-set}
```{tab-item} Windows
Install juliaup using `winget` from a terminal (Command Prompt or PowerShell):

    winget install --id Julialang.juliaup

Alternatively, install from the [Microsoft Store](https://www.microsoft.com/store/apps/9NJNWW8PVKMN).
```

```{tab-item} macOS / Linux
Run the following command from a terminal:

    curl -fsSL https://install.julialang.org | sh

Follow the on-screen prompts. When installation is complete, open a new terminal session (or run `source ~/.bashrc` / `source ~/.zshrc`) so that `juliaup` is available on your PATH.

macOS and Linux users can also install via Homebrew:

    brew install juliaup
```
````

Verify the installation was successful:
```
juliaup status
```

#### Step 2: Install and pin the tested Julia version

Install Julia `1.12.1` and set it as the default:
```
juliaup add 1.12.1
juliaup default 1.12.1
```

Confirm the active version:
```
julia --version
```

You should see `julia version 1.12.1`.

#### Step 3: Instantiate the Julia environment

> Navigate to the ReEDS-2.0 directory from the command line, then run:

Reviewer 2: This only works for a new install.  If you already have ReEDS set up and just want to upgrade, you need to delete `Manifest.toml`.  I think it's worth including a note like that here.  E.g.,

```suggestion
Navigate to the ReEDS-2.0 directory from the command line, then run (Note: if you already have a `Manifest.toml` file in the ReEDS respository, you should delete it before completing this step):
```

Author: I have added this in the troubleshooting section (https://github.com/ReEDS-Model/ReEDS/blob/7a5a43b0310e6a7235d01d333b1e89f094d9fe94/docs/source/setup.md?plain=1#L233) as I think it is better to keep this section for someone who is newly installing. Let me know if you'd like to have it here though

Reviewer 2: That works for me.  Thanks.

1. If Julia was installed outside of juliaup, uninstall it first: `winget uninstall julia`

1. Re-install Julia `1.12.1` via juliaup: `juliaup add 1.12.1 && juliaup default 1.12.1`

1. Open the Julia interactive command line: `julia`

1. Enter the Julia package manager by pressing `]`, then run the following commands:
    * `add Random123`
    * `registry add https://github.com/JuliaRegistries/General.git`
    * `instantiate`

1. Leave the package manager by pressing Backspace or Ctrl+C

1. Run the following commands to finish setup:
    * `import PRAS`
    * `import TimeZones`
    * `TimeZones.build()`

1. You can then leave the Julia command line by typing `exit()`

**macOS / Linux**

If you experience issues, try the following:

1. Update to a known-good Julia version via juliaup:
    ```
    juliaup add 1.12.1
    juliaup default 1.12.1
    ```

2. Run `julia` from the terminal to open the interactive command line

3. Run:
    ```julia
    import Pkg
    Pkg.add("PRAS")
    Pkg.add("TimeZones")
    ```

4. Exit Julia with `exit()`, then re-run:
    ```
    julia --project=. instantiate.jl
    ```

> 

Reviewer 3: Not sure if you want to add a note here or not, but I followed these steps and the only thing that resolved the error I was running into was deleting the Manifest.toml file and running the instantiate line again

#### Step 1: Install juliaup

````{tab-set}
```{tab-item} Windows
Install juliaup using `winget` from a terminal (Command Prompt or PowerShell):

    winget install --id Julialang.juliaup

Alternatively, install from the [Microsoft Store](https://www.microsoft.com/store/apps/9NJNWW8PVKMN).
```

```{tab-item} macOS / Linux
Run the following command from a terminal:

    curl -fsSL https://install.julialang.org | sh

Follow the on-screen prompts. When installation is complete, open a new terminal session (or run `source ~/.bashrc` / `source ~/.zshrc`) so that `juliaup` is available on your PATH.

macOS and Linux users can also install via Homebrew:

    brew install juliaup
```
````

Verify the installation was successful:
```
juliaup status
```

#### Step 2: Install and pin the tested Julia version

Install Julia `1.12.1` and set it as the default:
```
juliaup add 1.12.1
juliaup default 1.12.1
```

Confirm the active version:
```
julia --version
```

You should see `julia version 1.12.1`.

#### Step 3: Instantiate the Julia environment

> Navigate to the ReEDS-2.0 directory from the command line, then run:

Reviewer: No need to fix it everywhere; let's just clean up the new additions:

```suggestion
Navigate to the ReEDS directory from the command line, then run:
```
